# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../suspend/NokiaBooklet3G.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The usual guidance.

  - Rene Rebe <rene@exactcode.de>

--- suspend/whitelist.c.orig	2009-11-01 16:00:04.000000000 +0000
+++ suspend/whitelist.c	2009-11-01 16:02:20.000000000 +0000
@@ -291,6 +291,8 @@
 	{ "MAXDATA",			"PRO600IW",		"",	"", S3_BIOS|S3_MODE },
 	/* Hugo Costelha */
 	{ "NEC *",			"B7 *",			"",	"", VBE_SAVE }, 
+	/* Rene Rebe <rene@exactcode.de> */
+	{ "Nokia", 			"Nokia Booklet 3G",	"",	"", S3_BIOS },
 
 	/* Daniel Koester <koester@math.uni-augsburg.de> does not yet work from text mode */
 	{ "SAMSUNG ELECTRONICS CO., LTD.",	"R55S *",	"",	"", 0 },
